app/javascript/packs/src/settings/components/account/preferences/InputTimezone.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PropType from "prop-types";
+import { Button } from "react-bootstrap";
+import styled from "styled-components";
+import TimezonePicker from "react-bootstrap-timezone-picker";
+import "react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css";
+import { FormattedMessage } from "react-intl";
+
+import {
+  BORDER_LIGHT_COLOR,
+  PRIMARY_GREEN_COLOR,
+  PRIMARY_HOVER_COLOR
+} from "../../../../../app/constants/colors";
+
+const Wrapper = styled.div`
+  border: 1px solid ${BORDER_LIGHT_COLOR};
+  padding: 19px;
+  margin: 20px 0;
+
+  input {
+    margin-bottom: 3px;
+  }
+
+  .settings-warning {
+    margin-bottom: 15px;
+  }
+
+  .btn-primary {
```

Comment:
put this override in styles/main and give the color there if it apply to all btn-primary buttons :smile: